react/tic-tac-toe/src/config/api.js

```diff
@@ -0,0 +1,10 @@
+import { create } from 'apisauce';
+
+const api = create({
+  /* No toma el valor de API_BASE_URL */
```

Comment:
Try to avoid using code comments, and in case you are using them:
1. Write them in english since it's the standard.
2. Write them as `// TODO: Take a look back at this when something something something...` this way it'd be easier to find what has to be replaced/enhanced later on. 😸